test/e2e/tests/new-folder-flow/helpers/new-folder-flow.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Application, CreateFolderOptions, FolderTemplate } from '../../../infra/index.js';
+import { expect, test } from '../../_test.setup.js';
+
+export function addRandomNumSuffix(name: string): string {
+	return `${name}_${Math.floor(Math.random() * 1000000)}`;
+}
+
+export async function createNewFolder(app: Application, options: CreateFolderOptions) {
+	await test.step(`Create a new folder: ${options.folderName}`, async () => {
+		await app.workbench.newFolderFlow.createNewFolder(options);
+	});
+}
+
+export async function verifyFolderCreation(app: Application, folderName: string, folderTemplate: FolderTemplate) {
+	await test.step(`Verify folder created`, async () => {
+		await expect(app.code.driver.page.locator('#top-action-bar-current-working-folder')).toHaveText(folderName, { timeout: 60000 }); // this is really slow on windows CI for some reason
+
+		const consoleSymbol = folderTemplate === FolderTemplate.R_PROJECT ? '>' : '>>>';
```

Comment:
Should this maybe be skipped when folderTemplate is EMPTY_PROJECT?

Reply:
yes! This is now fixed